test/unit/app/models/edition/custom_lead_image_test.rb

```diff
@@ -4,28 +4,40 @@ class Edition::CustomLeadImageTest < ActiveSupport::TestCase
   include ActionDispatch::TestProcess
 
   def edition_with_custom_lead_image(options = {})
-    build(:draft_news_article, options)
-  end
+    edition = build(:draft_news_article, options)
+    image1 = build(:image)
+    image2 = build(:image)
 
-  test "reports that the first image is not available for adding inline" do
-    assert edition_with_custom_lead_image.image_disallowed_in_body_text?(1)
-  end
+    edition.stubs(:images).returns([image1, image2])
```

Comment:
What's the advantage of stubbing over setting up the association by doing `edition.images = [image1, image2]`? Or even creating the images first and passing them to the factory?

Reply:
thanks have sorted